Assets/SEE/UI/Window/TreeWindow/DesktopTreeWindow.cs

```diff
@@ -50,53 +56,116 @@ public partial class TreeWindow
         /// </summary>
         private static readonly GradientAlphaKey[] alphaKeys = { new(1, 0), new(1, 1) };
 
+        /// <summary>
+        /// The input field in which the user can enter a search term.
+        /// </summary>
+        private TMP_InputField SearchField;
+
+        /// <summary>
+        /// Orders the tree below the given <paramref name="orderBelow"/> node according to the graph hierarchy.
+        /// This needs to be called whenever the tree is expanded.
+        /// </summary>
+        /// <param name="orderBelow">The node below which the tree should be ordered.</param>
+        private void OrderTree(Node orderBelow)
+        {
+            Debug.Log("Ordering tree.");
+            int index = items.Find(CleanupID(orderBelow.ID)).GetSiblingIndex();
+            OrderTreeRecursive(orderBelow);
+
+            return;
```

Comment:
What's the purpose of this `return`?

Reply:
This return communicates to the reader of the code that the function ends here. This is relevant because local functions are below it – without the return, it is possible that additional statements occur after the local functions, but with the return, the local functions are clearly separated from the statements of the actual function.

Reply:
I see. Maybe that's worth a comment.